#CREATE request parameters
float64 angle_yaw
float64 angle_pitch
float64 time
--------------------------------
#CREATE response parameters
string resp

